examples/gcoap/server.c

```diff
@@ -100,6 +109,63 @@ static ssize_t _encode_link(const coap_resource_t *resource, char *buf,
     return res;
 }
 
+#if IS_USED(MODULE_PERIPH_RTC)
+static void _rtc_notify_observers(void *arg)
+{
+    (void)arg;
+    struct tm tm_now;
+    if (rtc_get_time(&tm_now)) {
+        DEBUG_PUTS("gcoap_server: RTC error");
+        return;
+    }
+    uint32_t t_now = rtc_mktime(&tm_now);
+    t_now = htonl(t_now);
+    size_t len;
+    uint8_t buf[sizeof(coap_hdr_t) + COAP_TOKEN_LENGTH_MAX + 1 + sizeof(t_now)];
+    coap_pkt_t pdu;
+    switch (gcoap_obs_init(&pdu, buf, sizeof(buf), &_resources[2])) {
```

Comment:
better add a comment that `&_resources[2]` is the `/rtc` resource - or better yet, add a function to get a resource by name.